<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddRCYVolunteers</class>
 <widget class="QDialog" name="AddRCYVolunteers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>220</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Last Name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_InsertPhoto">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>130</y>
     <width>181</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Insert Volunteer's Photo</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>320</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Profession</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_AddVolunteer">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>340</y>
     <width>121</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Volunteer</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_MobileNo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>150</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_SN">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>180</y>
     <width>150</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>First Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>491</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Volunteer's Information</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_FN">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>150</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_MN">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>180</y>
     <width>150</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ID">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>150</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>220</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Middle Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>270</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Mobile No.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Maab ID</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
